hutch_python/cli.py

```diff
@@ -28,3 +32,19 @@ def setup_cli_env():
 
     # Load objects from the configuration file
     return load(args.cfg)
+
+
+def hutch_ipython_embed():
+    """
+    This is very hacky, but I couldn't find a better way to adjust the shell
+    from a call to embed.
+    """
+    config = load_default_config()
+    config.InteractiveShellEmbed = config.TerminalInteractiveShell
+    frame = sys._getframe(1)
+    shell = InteractiveShellEmbed.instance(_init_location_id='%s:%s' % (
+        frame.f_code.co_filename, frame.f_lineno), config=config)
+    init_ipython_logger(shell)
+    shell(header=u'', stack_depth=2, compile_flags=None,
+          call_location_id='%s:%s' % (frame.f_code.co_filename,
+                                      frame.f_lineno))
```

Comment:
This scares me a little bit. Largely because I have no idea what it is doing. Let's leave it in, but I might be quick to blame it if funky stuff starts happening in our IPython windows.

Reply:
copy pasted from ipython and then removed all the unneeded lines